### DISCUSSION TITLE
Ellipsis in JSON arrays

### DISCUSSION THREAD

**Discussion by marian-lambert** [josdejong/jsonrepair#127] [Ideas]
Hi there,

not sure if this is something you want to include in this library but in my LLM testing, i saw multiple instances of LLMs generating arrays with ellipsis, e.g.:
`["123", "456", ...]`
This obviously fails to parse to JSON. Solution could be to just remove the ...:
`["123", "456", ...]` -> `["123", "456"]`

What do you think @josdejong ?

**Comment by josdejong** [comment]
Nice idea, thanks!

Same for objects I think, like `{"a": 2, "b": 3, ...}`

**Comment by josdejong** [comment]
Implemented now in `v3.8.0`
